### PR TITLE
feat(Task-4): As a Developer, I want to have the openWeatherMap api configured

### PR DESCRIPTION
**Pull request by jLagosCarrera**

### CODE REVIEW

File: src/app/app.component.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import template from './app.component.html';
+import './app.component.css';
+
+AppComponentController.$inject = ['$scope', 'weatherService'];
+
+function AppComponentController($scope, weatherService) {
```
Comment (eloydotco) [inline src/app/app.component.js]: It's a good practice create another file for the controller -> `app.controller.js`

File: src/app/app.module.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import angular from 'angular';
+import appComponent from './app.component';
+
+const app = angular.module('app', []);
+
+app.component('app', appComponent);
+app.factory('weatherService', ['$http', function ($http) {
```
Comment (eloydotco) [inline src/app/app.module.js]: Another good practice is create a new file for each new service, and put all on a `services` folder. Very good for write maintainable code

**Review by eloydotco — CHANGES_REQUESTED** [review]
Solve conflicts also

File: src/app/app.component.js (hunk 1 above)
Comment (gbtec-ext-jorgevillar) [reply src/app/app.component.js]: +1 to Eloy and in plus of that, create it in the EcmaScrip2015 way (with a class)
image

**Review by gbtec-ext-jorgevillar — CHANGES_REQUESTED** [review]

File: src/app/app.module.js [hunk 3]
```diff
@@ -13,4 +13,21 @@ export default angular.module('app', [uiRouter, angularanimate, angulararia, ang
...
+    
+        const request = {
+            method: 'GET',
+            url: URL,
+            params: {
+                q: 'Vigo',
+                mode: 'json',
+                units: 'metric',
+                lang: 'en',
+                appid: '37499dd44886aa89565abd094ecbfc8c'
```
Comment (diegopf) [inline src/app/app.module.js]: This repository is public, isn't? :) You must not commit such things as passwords or apikeys into public repositories. You can take a look at https://webpack.js.org/plugins/define-plugin/

**Review by diegopf — CHANGES_REQUESTED** [review]

**Comment by jLagosCarrera** [comment]
All requested changes done

**Review by eloydotco — APPROVED** [review]

**Review by diegopf — APPROVED** [review]

**Pull request merged**